joy2twist/src/main.cpp

```diff
@@ -0,0 +1,26 @@
+#include <iostream>
+
+#include "joy2twist/joy2twist_node.hpp"
+
+using namespace joy2twist;
+
+int main(int argc, char * argv[])
+{
+  ros::init(argc, argv, "joy2twist_node");
+
+  auto private_nh = std::make_shared<ros::NodeHandle>("~");
+  auto nh = std::make_shared<ros::NodeHandle>();
+  Joy2TwistNode joy2twist_node(private_nh, nh);
```

Comment:
That's the situation I tried to avoid by creating new node handles in class. Now it is necessary to pass two node handles, which looks worse in my opinion. But we already had discussion about it and came to conclusion that passing node handles is better, so it's ok.